### PR TITLE
US-3.12: Fix task drawer issues

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskDrawer.vue [hunk 1]
```diff
@@ -141,16 +141,64 @@
...
+                          style="color: #578DC9;font-weight: bold"
+                          color="#578DC9"
+                          v-html="item.title"/>
+                        <span>{{ item.comment }}</span>
+                        <v-btn 
+                          id="reply_btn" 
+                          text 
+                          small
+                          color="primary"
+                          @click="openEditor()">Reply</v-btn>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskDrawer.vue]: All labels should be internationalized

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/TaskDrawer.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/CkeditorVue.vue [hunk 2]
```diff
@@ -18,10 +18,17 @@
...
+        type: String,
+        default: ''
+      }
+    },
     data: function () {
       return {
-        editorData: '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>',
+        editorData: '',
         editorConfig: {
+          placeholder:'Add your comment...',
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/CkeditorVue.vue]: Can you internationalize this label ?

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/tasks/components/CkeditorVue.vue]: Doen

**Pull request merged**